scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -339,7 +389,8 @@
            bearer-token-refresh-seconds
            namespace
            scan-frequency-seconds
-           max-pods-per-node]
+           max-pods-per-node
+           synthetic-pods]
```

Comment:
Nitpick: rename to `synthetic-pods-config`

Reply:
Will do, thanks

Reply:
Actually, I think we want to keep this one as is. This is the key from the config, so it's nice for it to just be `:synthetic-pods`.